addon/ActivityStreams.js

```diff
@@ -227,6 +172,117 @@ ActivityStreams.prototype = {
     });
   },
 
+  _setUpPageWorker(store) {
+    this._pageWorker = new PageWorker({store});
+    this._pageWorker.connect();
+    this._refreshAppState();
```

Comment:
What do you think abt putting `buildAsyncCache` and `refreshAppState` in their own separate method? since they are both kind of related to initializing the data for the app